README.md

```diff
@@ -305,6 +305,68 @@ value, if available. If there is no fallback value, the token will be omitted
 from the JavaScript objects. In general, if you want to share a value between
 CSS and JavaScript, you should use simple strings or numbers.
 
+### Width Based Media Queries
+
+Gesso uses custom mixins to specify viewport width based media queries:
+`breakpoint`, `breakpoint-max` and `breakpoint-min-max` for min-width,
+max-width or a combination of both, respectively. Each mixin takes one or two
+width parameters, which can be a straight value (e.g., 800px, 40em) or a design
+token value called using the `gesso-breakpoint` function (e.g.,
+`gesso-breakpoint(tablet-lg)`).  The `breakpoint-max` and `breakpoint-min-max`
+mixins can also take an optional parameter to subtrax one pixel from the
+max-width value, which can be useful when using the Gesso breakpoint token
```

Comment:
Suggest replacing with `which can be useful when you want your query to go up to the value but not to include it, such as when using the Gesso breakpoint token values`.